content/20.pharmaceuticals.md

```diff
@@ -714,12 +714,27 @@ Specifically, IFNs I (IFN-𝛼 and 𝛽) and II (IFN-𝛾) induce the expression
 Among these IFNs, IFN-𝛽 has already been found to strongly inhibit the replication of other coronaviruses, such as SARS-CoV-1, in cell culture, while IFN-𝛼 and 𝛾 were shown to be less effective in this context [@doi:10.1016/j.jcv.2003.11.013].
 There is evidence that patients with higher susceptibility to ARDS indeed show deficiency in IFN-𝛽.
 For instance, infection with other coronaviruses impairs IFN-𝛽 expression and synthesis, allowing the virus to escape the innate immune response [@doi:10.1016/j.virusres.2014.07.024].
-On March 18 2020, Synairgen plc received approval to start a phase II trial for SNG001, an IFN-𝛽-1a formulation to be delivered to the lungs via inhalation.
-SNG001 was already shown to be effective in reducing viral load in an _in vivo_ model of swine flu and _in vitro_ models of other coronavirus infections [@synairgen-SNG001].
+On March 18 2020, Synairgen plc received approval to start a phase II trial for SNG001, an IFN-𝛽-1a formulation to be delivered to the lungs via inhalation [@clinicaltrials:NCT04385095].
+SNG001, which contains recombinant interferon beta-1a, was already shown to be effective in reducing viral load in an _in vivo_ model of swine flu and _in vitro_ models of other coronavirus infections [@synairgen-SNG001].
 In July, a press release from Synairgen stated that SNG001 reduced progression to ventilation in a double-blind, placebo-controlled, multi-center study of 101 patients with an average age in the late 50s [@url:https://www.synairgen.com/wp-content/uploads/2020/07/200720-Synairgen-announces-positive-results-from-trial-of-SNG001-in-hospitalised-COVID-19-patients.pdf].
-They also reported that patients in the treatment group showed greater recovery and lower breathlessness.
-However, given that this information was released in a press release rather than in a manuscript and thus cannot be thoroughly reviewed, these findings should be considered preliminary.
-Additionally, the WHO Solidarity trial reported no significant effect of IFN-𝛽1a on patient survival during hospitalization [@doi:10.1056/NEJMoa2023184].
+These results were subsequently published in November 2020 [@doi:10/ghjzm4].
+The study reports that the participants were assigned at a ratio of 1:1 to receive either SNG001 or a placebo that lacked the active compound, by inhalation for up to 14 days.
+The primary outcome they assessed was the change in patients' score on the WHO Ordinal Scale for Clinical Improvement (OSCI) at day 15/16.
```

Comment:
Just to clarify, does `15/16` mean that the patients were assessed on day 15 or 16 (so 1 or 2 days after treatment ended)? Or is this saying day 15 of the 16-day trial?

Reply:
```suggestion
The primary outcome they assessed was the change in patients' score on the WHO Ordinal Scale for Clinical Improvement (OSCI) at trial day 15 or 16.
```
It was day 15 or 16, aka a day or two after treatment ended -- so I think the answer to your question is both? haha